examples/examples_load_optimization/001_load_ti_and_voc.py

```diff
@@ -0,0 +1,131 @@
+"""Example: Load turbulence intensity
```

Comment:
The file name was confusing to me at first because I was expecting this example to "load" or "import" Tis and VOCs. What do you think about changing to "001_investigate_load_ti_and_voc"?

Reply:
Ok should be a clearer now